### ISSUE TITLE
Considering moving to RTD

### ISSUE THREAD

**Issue by de-sh** [HTTP-APIs/http-apis.github.io#43]
Would the community agree with moving the documentation to a more "Docmentation specific" website framework, such as [ReadTheDocs](https://readthedocs.org)(Also built in Python) for managing the docs instead of using the custom jekyll instance currently in use.

**Comment by chrizandr** [comment]
We might also consider using Pelican to generate static pages. I use it in one of my projects and I found it to be really helpful.
https://www.fullstackpython.com/pelican.html

**Comment by xadahiya** [comment]
I don't think there is any need to switch our stack. Pelican doesn't provide much as compared to Jekyll. See comparison between the two here https://www.slant.co/versus/1006/1011/~jekyll_vs_pelican

**Comment by de-sh** [comment]
I also think we don't need a change of static generator, but we can try out what [writethedocs.org](https://writethedocs.org) has done with [rtd](readthedocs.org)

**Comment by Guttz** [comment]
I agree with @de-sh, would be good to have a bit more structure in our docs, perhaps a search tool and for sure a side navigation along with the documentation like https://www.tensorflow.org/guide/
and https://svelte.dev/docs.

Ran into a jenkyll doc theme which might be useful: https://idratherbewriting.com/documentation-theme-jekyll/

**Comment by de-sh** [comment]
If there is a good contender for a static-gen to move the current source to, it seems like [ABlog](https://github.com/sunpy/ablog) is one, it is built with Sphinx and is in Python, so we could pretty easily show our legacy of having worked with the PSF.

**Comment by de-sh** [comment]
If we are still going to use Jekyll it might be a good idea to also venture out of using just and only GitHub pages. As a community, I think we are too dependent on this platform for all our requirements, I know it is a controversial opinion, but maybe we could consider using something like netlify.com for hosting our documentation. We can add a hook that deploys on each update of `master`.

Essentially we could drop `github.io` in the name of the documentation and still be able to use GitHub to do the source control.

**Comment by shravandoda** [comment]
We can also take a look at Hugo. It's written in Golang and I found it quite useful

**Comment by de-sh** [comment]
I too use Hugo 😃

But at the moment the issue is with how to build the docs to be a presentation, an entry-point into building with the hydraecosystem. Jekyll is good enough as @xadahiya points out, we have to create a theme that can be better suited for documentation.
